--- retis-1.4.0-base/Cargo.toml	2024-04-24 15:17:32.000000000 +0200
+++ retis-1.4.0/Cargo.toml	2024-05-23 19:41:47.158061036 +0200
@@ -47,7 +47,7 @@
 pcap = "2.0"
 pcap-file = "2.0"
 plain = "0.2"
-pnet = "0.34"
+pnet_packet = "0.34"
 rbpf = {version = "0.2", optional = true}
 regex = "1.7"
 retis-derive = {version = "1.4", path = "./retis-derive"}
Only in retis-1.4.0/src/core/probe/kernel/bpf: .out
Only in retis-1.4.0/src/core/probe/user/bpf: .out
Only in retis-1.4.0/src/module/ct/bpf: .out
Only in retis-1.4.0/src/module/nft/bpf: .out
Only in retis-1.4.0/src/module/ovs/bpf: .out
Only in retis-1.4.0/src/module/skb/bpf: .out
--- retis-1.4.0-base/src/module/skb/bpf.rs	2024-04-24 15:17:32.000000000 +0200
+++ retis-1.4.0/src/module/skb/bpf.rs	2024-05-23 19:42:22.116242923 +0200
@@ -7,7 +7,7 @@
 use std::str;
 
 use anyhow::{anyhow, Result};
-use pnet::packet::{
+use pnet_packet::{
     arp::ArpPacket, ethernet::*, icmp::IcmpPacket, icmpv6::Icmpv6Packet, ip::*, ipv4::*, ipv6::*,
     tcp::TcpPacket, udp::UdpPacket, Packet,
 };
Only in retis-1.4.0/src/module/skb_drop/bpf: .out
Only in retis-1.4.0/src/module/skb_tracking/bpf: .out
Only in retis-1.4.0/src: .out
Only in retis-1.4.0: target
