--- a/retis/Cargo.toml
+++ b/retis/Cargo.toml
@@ -22,7 +22,6 @@ maintenance = { status = "actively-developed" }
 test_bindgen_layout = []
 test_cap_bpf = []
 benchmark = []
-debug = ["dep:rbpf"]
 
 [dependencies]
 anyhow = "1.0"
@@ -47,7 +46,6 @@ pcap = "1.3"
 pcap-file = "2.0"
 plain = "0.2"
 pnet_packet = "0.34"
-rbpf = {version = "0.2", optional = true}
 regex = "1.7"
 retis-derive = {version = "1.4", path = "../retis-derive"}
 serde = {version = "1.0", features = ["derive"]}
